Cirumflex_Combined.ino

```diff
@@ -1,12 +1,19 @@
 //Fixed colors set as globals. 03-Pattern fades between them.
 const int RANDOMnoisePIN = A5; // for random seed
 const int BRIGHTNESS = 1;      // Value pot pin number
-const int SLOWNESS = 3;        // Pot pin number; 0 is fast, 1023 is slow. 
+const int SLOWNESS = 3;        // Pot pin number; 0 is fast, 1023 is slow.
+
+// Which LED interface?
+#include "PWM_NativePin.h"
+PWM_NativePin_MEGA PWM;
```

Comment:
Control pwm pins through `PWM`. We can substitute a pwm controller for native pins, later.
The pattern is:
'''
#include "PWM_xxxxx.h"
PWM_xxxxx PWM; // might be a more specific xxxx
'''

Reply:
I'm not clear what "Native pins" means. But it doesn't look too hard to straighten out.

Reply:
"Native" means the pins on the arduino, e.g. 9 is usually a pwm pin, etc. Is there a better term for "pins on the arduino" as opposed to "pins on some attached device like the TLC"?